Framework/API/src/FileFinder.cpp

```diff
@@ -185,7 +191,7 @@ const Kernel::InstrumentInfo FileFinderImpl::getInstrument(const string &hint) c
  * @return A pair of instrument name and run number
  */
 std::pair<std::string, std::string> FileFinderImpl::toInstrumentAndNumber(const std::string &hint) const {
-  // g_log.debug() << "toInstrumentAndNumber(" << hint << ")\n";
+  g_log.debug() << "toInstrumentAndNumber(" << hint << ")\n";
```

Comment:
Uhm, I uncommented it because when I was trying to understand what the code did it was useful to have this output to the console. But since it was commented out before I can also take it out?

Reply:
I think its fine to leave in that case 👍